packages/components/src/range-control/styles/range-control-styles.ts

```diff
@@ -48,7 +48,6 @@ const wrapperMargin = ( { marks, __nextHasNoMarginBottom }: WrapperProps ) => {
 };
 
 export const Wrapper = styled.div< WrapperProps >`
-	color: ${ COLORS.blue.medium.focus };
```

Comment:
This line wasn't needed because `color` was being assigned again a few lines later (`${ wrapperColor };` )

Reply:
I wrote this down in the commit message so I don't forget, and then forgot to write it as a comment 😂 Sorry to make you sleuth!